packages/formation/sass/site/_m-vet-nav.scss

```diff
@@ -216,7 +216,7 @@ body.va-pos-fixed {
     }
   }
 
-  &[aria-haspopup]:not([aria-expanded="true"]) {
+  &[aria-expanded='false'] {
```

Comment:
Removing the `[aria-haspopup]` was why I opted to do major version bumps. I don't anticipate breaks because of higher selector specificity, but it's a low cost way of being eyes up.

Reply:
On further discussion, we are going to call this a `minor` bump. The Formation codebase doesn't use `aria-haspopup` anywhere else for CSS styling. Did a full-text search for the strings "aria-haspopup" and "haspopup" in the Formation code base locally, and came up with no results.